assets/javascripts/kitten/components/date-picker/index.js

```diff
@@ -0,0 +1,280 @@
+import React, { Component } from 'react'
+// Doc available here: http://react-day-picker.js.org/docs/getting-started
+import DayPickerInput from 'react-day-picker/DayPickerInput'
+import dateFnsFormat from 'date-fns/format'
+import dateFnsParse from 'date-fns/parse'
+import styled, { css } from 'styled-components'
+import COLORS from '../../constants/colors-config'
+import TYPOGRAPHY from '../../constants/typography-config'
+import { TextInputWithUnit } from '../../components/form/text-input-with-unit'
+import {
+  WEEKDAYS_SHORT,
+  MONTHS,
+  WEEKDAYS_LONG,
+  FIRST_DAY_OF_WEEK,
+  LABELS,
+  FORMAT,
+} from './date-picker-config'
+import { Navbar } from './components/navbar'
+
+const borderSize = '2px'
+const cellSize = '50px'
+
+const fontSize = css`
+  font-size: ${TYPOGRAPHY.fontSize / TYPOGRAPHY.scaleMultiplier}rem;
+`
+
+const StyledDatePicker = styled.div`
+  box-sizing: border-box;
+
+  .DayPickerInput-Overlay {
+    box-shadow: none;
+    margin-top: 18px;
+    outline: none;
+    /* Arrow's top header */
+    &:before {
+      content: '';
+      width: 0;
+      height: 0;
+      border-left: 9px solid transparent;
+      border-right: 9px solid transparent;
+      ${({ styles }) => css`
+        border-bottom: 8px solid ${styles.header.backgroundColor};
+      `}
+      position: absolute;
+      top: -8px;
+      left: 15px;
+    }
+  }
+  /* Header */
+  .DayPicker-Caption {
+    text-align: center;
+    height: 70px;
+    margin: -2px -32px 0;
+    overflow: hidden;
+
+    ${({ styles }) => css`
+      background-color ${styles.header.backgroundColor};
+      color: ${styles.header.color};
+    `}
+
+    & div {
+      padding: 23px 0;
+      margin: 0 63px;
+      ${fontSize}
+    }
+  }
+
+  .DayPicker-Weekday {
+    ${TYPOGRAPHY.fontStyles.regular}
+
+    ${({ styles }) => css`
+      color: ${styles.weekdaysColor};
+    `}
+  }
+
+  .DayPicker-Weekday,
+  .DayPicker-Day {
+    padding: 0;
+    width: ${cellSize};
+    height: ${cellSize};
+    vertical-align: middle;
+    box-sizing: border-box;
+    ${fontSize}
+    outline: none;
+  }
+
+  .DayPicker-Day {
+    ${({ styles }) => css`
+      border: solid ${styles.borderColor} ${borderSize};
+    `}
+    border-radius: 0;
+
+    &.DayPicker-Day--outside {
+      border: 0;
+    }
+  }
+
+  .DayPicker-Day--today {
+    ${TYPOGRAPHY.fontStyles.regular}
+    ${({ styles }) => css`
+      color: ${styles.day.today.color};
+    `}
+  }
+
+  .DayPicker-Day--selected {
+    &:not(.DayPicker-Day--disabled) {
```

Comment:
Dommage, il n'y a qu'avec Safari qu'on peut faire `&:not(.DayPicker-Day--disabled, .DayPicker-Day--outside)`. En revanche, peut-être que ça allège un peu si tu fais :
```suggestion
      &:not(.DayPicker-Day--disabled):not(.DayPicker-Day--outside) {
```

Reply:
J'ai opté pour ce layout pour la lisibilité. Pour l'imbrication de class, je n'ai pas eu le choix car c'est la css utilisée par le composant `react-day-picker`.

Reply:
Sinon en terme de rendu c'est la même chose :) :
![image](https://user-images.githubusercontent.com/523306/53355142-aa4a9380-3928-11e9-8116-cb9096d0a6bd.png)